src/main/java/uk/gov/justice/hmpps/prison/api/resource/ServiceAgencySwitchesResource.kt

```diff
@@ -32,7 +32,7 @@ class ServiceAgencySwitchesResource(private val service: ServiceAgencySwitchesSe
     ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
   )
   @Operation(summary = "Retrieve a list of prisons switched on for the service code")
-  @PreAuthorize("hasRole('SERVICE_AGENCY_SWITCHES')")
+  @PreAuthorize("hasAnyRole('SERVICE_AGENCY_SWITCHES', 'SERVICE_AGENCY_SWITCHES__RO')")
```

Comment:
would be better for new roles to follow the convention of being prefixed by Prison API, so
```suggestion
  @PreAuthorize("hasAnyRole('SERVICE_AGENCY_SWITCHES', 'PRISON_API__SERVICE_AGENCY_SWITCHES__RO')")
```